src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailHandler.java

```diff
@@ -39,22 +40,39 @@ public MailHandler(Properties props, Authenticator auth) throws MessagingExcepti
         connectToStore();
```

Comment:
I solved the problem, but not the way you wanted, because the other way i got lots of spaghetti code. Now the store has the same lifetime as a message or the folder of the message.